test/formatters/deployResultFormatter.test.ts

```diff
@@ -58,7 +59,7 @@ describe('DeployResultFormatter', () => {
     it('should return expected json for a success', async () => {
       const deployResponse = JSON.parse(JSON.stringify(deployResultSuccess.response)) as DeployCommandResult;
       const expectedSuccessResults = deployResultSuccess.response as DeployCommandResult;
-      const formatter = new DeployResultFormatter(logger, ux, {}, deployResultSuccess);
+      const formatter = new DeployResultFormatter(logger, ux as UX, {}, deployResultSuccess);
```

Comment:
we can't DRY this up because ux is type `stubInterface` and not a real `UX` correct?